pkg/networkengine/vpndriver/wireguard/wireguard.go

```diff
@@ -561,29 +486,3 @@ func parseSubnets(subnets []string) []net.IPNet {
 	}
 	return nets
 }
-
-func (w *wireguard) ensureRavenSkipNAT(centralGw *types.Endpoint, connection *vpndriver.Connection) error {
```

Comment:
why delete this function?

Reply:
Seems I missed the 221 a303d0895ea9a7b9fbfb2aaf2a6be6d24cc8a submitted，I has fix it